cupy/cuda/cupy_cudnn.h

```diff
@@ -118,6 +118,76 @@ cudnnStatus_t cudnnSetConvolutionNdDescriptor_v3(
 #define cudnnSetPooling2dDescriptor_v3 cudnnSetPooling2dDescriptor
 #define cudnnSetPoolingNdDescriptor_v3 cudnnSetPoolingNdDescriptor
 
+cudnnStatus_t cudnnDeriveBNTensorDescriptor(
+         cudnnTensorDescriptor_t derivedBnDesc,
+         const cudnnTensorDescriptor_t xDesc,
+         cudnnBatchNormMode_t mode) {
```

Comment:
Please define `cudnnBatchNormMode_t`.
Build is failed when CuPy uses lower than cuDNN v5.